staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -233,6 +233,9 @@ if grep -qw "lister" <<<"${GENS}"; then
 fi
 
 if grep -qw "informer" <<<"${GENS}"; then
+  CLIENTSET_PKG="${CLIENTSET_PKG_NAME:-clientset}"
```

Comment:
These lines exist above but maybe were not triggered.  Can we do this instead:

```diff
 diff --git a/staging/src/k8s.io/code-generator/generate-internal-groups.sh b/staging/src/k8s.io/code-generator/generate-internal-groups.sh
index db1f93f43ff..9676fac313b 100755
--- a/staging/src/k8s.io/code-generator/generate-internal-groups.sh
+++ b/staging/src/k8s.io/code-generator/generate-internal-groups.sh
@@ -120,6 +120,9 @@ for GVs in ${GROUPS_WITH_VERSIONS}; do
   done
 done
 
+CLIENTSET_PKG="${CLIENTSET_PKG_NAME:-clientset}"
+CLIENTSET_NAME="${CLIENTSET_NAME_VERSIONED:-versioned}"
+
 if grep -qw "deepcopy" <<<"${GENS}"; then
   # Nuke existing files
   for dir in $(GO111MODULE=on go list -f '{{.Dir}}' "${ALL_FQ_APIS[@]}"); do
@@ -187,9 +190,6 @@ if grep -qw "applyconfiguration" <<<"${GENS}"; then
 fi
 
 if grep -qw "client" <<<"${GENS}"; then
-  CLIENTSET_PKG="${CLIENTSET_PKG_NAME:-clientset}"
-  CLIENTSET_NAME="${CLIENTSET_NAME_VERSIONED:-versioned}"
-
   # Nuke existing files
   root="$(GO111MODULE=on go list -f '{{.Dir}}' "${OUTPUT_PKG}/${CLIENTSET_PKG}/${CLIENTSET_NAME}" 2>/dev/null || true)"
   if [ -n "${root}" ]; then
```

Reply:
> These lines exist above but maybe were not triggered. Can we do this instead:

Correct, but those are only invoked when generating client, if you invoke informer separately from client it won't. 
Also, that was the other option I was considering - updated.